docs/reference/requirement-sets/excel-preview-apis.md

```diff
@@ -16,6 +16,7 @@ The first table provides a concise summary of the APIs, while the subsequent tab
 
 | Feature area | Description | Relevant objects |
 |:--- |:--- |:--- |
+| Formula changed events | Track changes to formulas, including the source and type of event that caused a change. | [onFormulaChanged](/javascript/api/excel/excel.worksheet#onFormulaChanged)|
```

Comment:
That last column typically has names of objects instead of methods. I'm okay bucking the trend if you don't want to take the suggested change, though I'd recommend prefixing the event name with `Worksheet.` in that case.
```suggestion
| Formula changed events | Track changes to formulas, including the source and type of event that caused a change. | [Worksheet](/javascript/api/excel/excel.worksheet#onFormulaChanged)|
```

Reply:
I will change it to Worksheet.onFormulaChanged.